plugin/adsk/scripts/mayaUSDRegisterStrings.mel

```diff
@@ -209,6 +209,13 @@ global proc mayaUSDRegisterStrings()
     register("kImportMaterialConvPSLbl", "USD Preview Surface");
     register("kImportMaterialConvBlnLbl", "Blinn");
     register("kImportMaterialConvPhgLbl", "Phong");
+    register("kImportRelativeTexturesLbl", "Texture File Paths:");
+    register("kImportRelativeTexturesAnn", "<b/>Choose whether your texture files are written as relative or absolute paths\n" + 
```

Comment:
Why does this string have a <b/> at the front?

Reply:
Thanks @seando-adsk !